R/Shiny.R

```diff
@@ -42,8 +39,7 @@
 #' Launches a Shiny app that allows the user to explore the diagnostics
 #'
 #' @export
-launchDiagnosticsExplorer <- function(dataFolder = "data",
-                                      dataFile = "PreMerged.RData",
+launchDiagnosticsExplorer <- function(sqliteDbPath = "MergedCohortDiagnosticsData.sqlite",
```

Comment:
would this be considered breaking change? support to premerged file is gone?

Reply:
Yes, I don't think its a good idea to continue to support the PreMerged.RData file in newer versions of the app. Though it may be beneficial to allow some sort of conversion mechanism. However, I assumed that the zip files are the main results.

Reply:
Yes - ur assumption is correct. ZipFiles are the main results. So, based on that - i think this breaking change (i.e. not using premerged file) is ok